samtranslator/open_api/open_api.py

```diff
@@ -37,10 +37,10 @@ def __init__(self, doc):
         modifications on this copy.
 
         :param dict doc: OpenApi document as a dictionary
-        :raises ValueError: If the input OpenApi document does not meet the basic OpenApi requirements.
+        :raises InvalidDocumentException: If the input OpenApi document does not meet the basic OpenApi requirements.
         """
         if not OpenApiEditor.is_valid(doc):
-            raise ValueError(
+            raise InvalidDocumentException(
```

Comment:
Thank you! We should probably do another pass and stop throwing generic python errors/exceptions.